Sources/Moya/Response.swift

```diff
@@ -38,14 +42,17 @@ public extension Response {
         return self
     }
 
+    /// Filters out responses of a specific status code, generating errors when others are encountered.
```

Comment:
These just return `an Error`, right? Not multiple. Might want to make that more clear.